### ISSUE TITLE
Nuitka with pyVmomi all memory take by gcc

### ISSUE THREAD

**Issue by stravassac** [Nuitka/Nuitka#353]
Nuitka with pyVmomi crash, all memory need by cc1 
OOM killer destroy it
kernel: [ 8152.785701] oom_reaper: reaped process 22768 (cc1), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB

*  Nuitka version, full Python version and Platform (Windows, OSX, Linux ...)

python -m nuitka --version
0.6.4rc2
Python: 3.6.7 (default, Oct 22 2018, 11:32:17) 
Executable: /home/stravassac/virtualenvs/test/bin/python
OS: Linux
Arch: x86_64

gcc (Ubuntu 7.3.0-27ubuntu1~18.04) 7.3.0

*  How did you install Nuitka and Python (pip, anaconda, deb, rpm, from source,
   what is a virtualenv ...), this is very important usually.
python is from Ubuntu 18.04
```
pip freeze
certifi==2019.3.9
chardet==3.0.4
idna==2.8
Nuitka==0.6.4rc2
pkg-resources==0.0.0
pyvmomi==6.7.1.2018.12
requests==2.21.0
six==1.12.0
urllib3==1.24.1
```
*  If possible please supply a [Short, Self Contained, Correct, Example](http://sscce.org/)
   that demonstrates the issue i.e a small piece of code which reproduces
   the issue and can be run with out any other (or as few as possible)
   external dependencies.
nice -n 19 python -m nuitka test.py
```
import sys
import os
from pyVim import connect
from pyVmomi import vim, vmodl

def main():
    print(os.getpid())


print(sys.argv)
if __name__ == "__main__":
    main()
```

**Comment by kayhayen** [comment]
I managed to reproduce it, with OOM killer and all, on a 16GB+16GB swap machine. Now that wasn't fun, and I was pretty confused, when my work environment slowed down, and eventually blocked. I should have been warned. :)

**Comment by kayhayen** [comment]
So, `pyVmomi.ServerObjects` contains a lot of code, very specific one, that is breaking all the scalability of Nuitka and gcc obviously. I am going to blacklist it from compilation as "unworthy" namespace, the next pre-release should be working for you then.

**Comment by stravassac** [comment]
Sorry for your slowdown, I forgot to tell you to disable your swap (swapoff -a) for have a less freeze (1 minute for me, before oomkiller)

**Comment by stravassac** [comment]
> I managed to reproduce it, with OOM killer and all, on a 16GB+16GB swap machine. Now that wasn't fun, and I was pretty confused, when my work environment slowed down, and eventually blocked. I should have been warned. :)

**Comment by stravassac** [comment]
@kayhayen Thanks you for your quick reply and and quick fix, sorry again for your computer slowed down

**Comment by kayhayen** [comment]
Released as 0.6.4
